tests/test_browser.py

```diff
@@ -3129,9 +3129,16 @@ def post_build(*args):
   def test_sdl2_ttf(self):
     shutil.copy2(path_from_root('tests', 'freetype', 'LiberationSansBold.ttf'), self.get_dir())
     self.btest('sdl2_ttf.c', reference='sdl2_ttf.png',
-               args=['-O2', '-s', 'USE_SDL=2', '-s', 'USE_SDL_TTF=2', '--embed-file', 'LiberationSansBold.ttf'],
+               args=['-O2', '-s', 'USE_SDL=2', '-s', 'USE_SDL_TTF=2', '-s', 'USE_HARFBUZZ=1', '--embed-file', 'LiberationSansBold.ttf'],
```

Comment:
why is this needed - doesn't SDL2_ttf automatically add it?

Reply:
I'm not sure whether I undestood you correctly. Aren't these flags supposed to be passed to `emcc` when using ports? Are theses tests treated differently? (infact I removed the flag locally and the tests passed just fine)